jenkins/workers.py

```diff
@@ -0,0 +1,94 @@
+"""
+Use the jenkins queue and computer APIs to log info about workers.
+
+Usage:
+    `python jenkins/workers.py -j https://example.jenkins.com`
+
+Example Output:
+    [INFO] Counts of workers and queue length from Jenkins API:
```

Comment:
The timestamps are included in that job on jenkins (https://build.testeng.edx.org/job/edx-platform-test-subset/77858/console), so I was curious how splunk handled those ones.